lib/dialects/mssql/transaction.js

```diff
@@ -48,21 +45,19 @@ module.exports = class Transaction_MSSQL extends Transaction {
     );
   }
 
-  rollbackTo(conn, error) {
+  async rollbackTo(conn, error) {
     debug('%s: rolling backTo', this.txid);
-    return Bluebird.resolve()
-      .then(() =>
-        this.query(conn, `ROLLBACK TRANSACTION ${this.txid}`, 2, error)
-      )
-      .then(() => this._rejecter(error));
+    await this.query(conn, `ROLLBACK TRANSACTION ${this.txid}`, 2, error);
+
+    this._rejecter(error);
```

Comment:
This appears to replicate the old logic...

... BUT: It's a bit surprising that `_this.rejector(error)` only gets invoked when `this.query(..)` succeeds.  We should investigate this further once we've merged this PR.

Reply:
Yeah... this behavior seems to be inconsistent across `Transaction` implementations.  So, we can follow up on this once we resume the `Transaction` refactoring.

Reply:
we can create issue where track full review of code